include/navigation_plotting/plot.h

```diff
@@ -4,11 +4,11 @@
 #include <vector>
 
 namespace BoBRobotics {
-namespace Navigation {
+namespace NavigationPlotting {
 
 //! Plot a line graph of a single RIDF
 void
 plotRIDF(const std::vector<float> &differences);
 
-} // Navigation
+} // NavigationPlotting
```

Comment:
It might make more sense just to stick this in the ``Viz`` module rather than making a new one

Reply:
:+1: